pkg/operation/care/health.go

```diff
@@ -87,9 +87,10 @@ func (h *Health) Check(
 	ctx context.Context,
 	thresholdMappings map[gardencorev1beta1.ConditionType]time.Duration,
 	healthCheckOutdatedThreshold *metav1.Duration,
+	managedResourceProgressingThreshold *metav1.Duration,
```

Comment:
Should we make this also part of the struct? If not, then can we harmonize the above case too?